docs/API.md

```diff
@@ -54,7 +54,7 @@ createOnEnter({
   ?authenticatingSelector: (state: Object, nextState: Object) => boolean,
   ?AuthenticatingComponent: ReactClass | ReactFunctionalComponent | string,
   ?wrapperDisplayName: string,
-  ?allowRedirectBack: string | (state: Object, nextState: Object) => boolean,
+  ?allowRedirectBack: string | (props: Object, redirectPath: String) => boolean,
```

Comment:
This should actually be `(nextState: Object, redirectPath: String)` - https://github.com/mjrussell/redux-auth-wrapper/blob/96414c997c27478d3be12b7ed133e99e65996338/src/history3/redirect.js#L51

Reply:
Interesting. I was confused by this - https://github.com/mjrussell/redux-auth-wrapper/blob/aa1ea4bdde2dc5b25b9f12c247614444c485e5db/test/redirectBase-test.js#L214